client/cmd/job/run_input.go

```diff
@@ -127,7 +134,10 @@ func (j *jobRunInputCommand) RunE(_ *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("request failed for job %s: %w", jobName, err)
 	}
```

Comment:
sendJobRunInputRequest should be in the Retry, WriteInstanceResponse is local no need to have retries for that